### ISSUE TITLE
bug: Exception thrown: 'System.FormatException' in System.Private.CoreLib.dll

### ISSUE THREAD

**Issue by murataka** [Dartanlla/OWS#57]
OwsInstanceManagement is continuously throwing this exception, likely every second.

`Exception thrown: 'System.FormatException' in System.Private.CoreLib.dll`

Mysql , UE5 is used . .net 6.0.7

Everything is fully configured and all the functionality is working. So , that is **not an urgent case** , but still It hides the real misconfiguration problems -  for the project, and it gets much harder to configure everything. Also it is very hard to find where that bug appears. 

If you get this error , and your project does not work , be sure to not have other misconfiguration problems.

**Comment by CanisHelix** [comment]
Does this only happen with MySQL version or also with MSSQL?

**Comment by murataka** [comment]
I will also test with posgresql , or even  try to find the root cause and update you. I do not have mssql .

**Comment by CanisHelix** [comment]
A few questions:

1) Does this happen even prior to loading a 'worldserver' via the dotnet command line?
2) Is there anything else around the exception that might indicate a line number?

If you are running with docker-compose can you share your .env file (replacing IP/Passwords with dummy etc).

I'm running this in Docker (directly rather than docker-compose) with the following environment variables set and do not see the same issues (but I'm not running a world server currently):

![image](https://user-images.githubusercontent.com/7948640/182501936-119c462c-59b6-4a35-8880-2d2ae418fd71.png)

**Comment by murataka** [comment]
Does this happen even prior to loading a 'worldserver' via the dotnet command line?
Yes
Is there anything else around the exception that might indicate a line number?
No. 

I'm running this in Docker (directly rather than docker-compose) with the following environment variables set and do not see the same issues (but I'm not running a world server currently) (host names blanked out):

Only from visual studio logs at  debug mode .

**Comment by CanisHelix** [comment]
Strange, could you upload the .env file so we can check it out? It could be an issue, and it's the only thing I can think of at this time.

**Comment by murataka** [comment]
Let me find out what's happening, maybe I will be able to solve that. If you can not replicate , maybe it is because of platform differences..

**Comment by CanisHelix** [comment]
Also, if you prefer, I am over on the official discord server too and I can also help out there.

**Comment by CanisHelix** [comment]
If you are using the latest version of the source code. Have you ran all of the scripts in https://github.com/Dartanlla/OWS/tree/main/Databases/MySQL/UpdateScripts against the database? These are not run automatically and users/devs need to run these against the databases manually at the moment. Just thinking this could be due to a Column change made a few weeks ago.

**Comment by murataka** [comment]
> 

sure, else it does not work already.

**Comment by murataka** [comment]
@CanisHelix , I moved to postgresql again and it is same again. 
<img width="1680" alt="Screen Shot 2022-08-04 at 19 14 56" src="https://user-images.githubusercontent.com/5358438/182897635-9acea029-ee0d-4044-abe3-4a2fcd3495d9.png">

SELECT * FROM public.owsversion
 20220613

* It does not differ if I have started the instancelauncher or not. 

* I am on a macbook m1 , and .net 5 does not work, so I upgraded the dependency libraries to 6.0.7.


I needed also to change DbQuery to DbSet, however , they are not exactly same. check this (possibly the root cause of it on .net 6.0.7 ) :https://github.com/dotnet/efcore/issues/15656
